battleship.py

```diff
@@ -216,7 +222,32 @@ def check_for_game_end_scenario(self):
             self.game_over = True
         elif(self.bomb_count <= 0):
             print('Game Over.You have depleted all of your remaining bombs.')
-            self.game_over = True         
+            self.game_over = True    
+
+
+class BoardPrinter:
+    lines = []
+    message =[]
+
+    def add_line(self, line):
+        self.lines.append(line)
+
+    def add_message_line(self, line):
+        self.message.append(line)
+
+    def write(self):
+        for _ in range(50): # 50 is arbitrary, but enough
```

Comment:
What is this range again?

Reply:
It's just a random number. I chose 50 so that way we have a buffer if we ever add more lines, those will get erased too. 

We could potentially figure out how many lines we are printing to the console but because we are using the pip package for user input this would be non-trivial. So the simplest solution is to just set this to a number larger than we will probably ever need